contract_show_sale/models/account_analytic_account.py

```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+
+
+class AccountAnalyticAccount(models.Model):
+    _inherit = 'account.analytic.account'
+
+    @api.multi
+    def _get_total_sale(self):
+        sale_model = self.env['sale.order']
+        for analytic in self:
+            fetch_data = sale_model.read_group(
```

Comment:
here Better to use `search_count` method.

```
@api.multi
    def _get_total_sale(self):
        sale_model = self.env['sale.order']
        for analytic in self:
            fetch_data = sale_model.search_count(
                [('project_id', '=', analytic.id),
                 ('state', 'not in', ['draft', 'sent', 'cancel'])],
                )
            analytic.total_sale = fetch_data
```

Reply:
mmm, doesn't it only give record count? I need total amount, not only record count